src/main/java/core/basesyntax/model/User.java

```diff
@@ -8,6 +8,17 @@ public class User {
     private String password;
     private Integer age;
 
+    public User() {
+
+    }
+
+    public User(Long id, String login, String password, Integer age) {
+        this.id = id;
+        this.login = login;
+        this.password = password;
+        this.age = age;
+    }
+
```

Comment:
Do we really need to initialize the `id` field when creating objects considering that there's a `setId ` method that gets called once we successfully add a user? Please check